.github/workflows/zoom-upload.yml

```diff
@@ -0,0 +1,46 @@
+name: Upload Zoom Recordings to YouTube
+
+on:
+  schedule:
+    # 10 minutes after every hour before noon on weekdays
+    # TODO: when we are ready to switch off Circle, run every hour instead of
+    # just in the morning.
+    - cron: "10 0-12 * * 1-5"
+
+  workflow_dispatch: {}
+
+  # XXX: This is for testing; do not merge
+  pull_request: {}
```

Comment:
Make sure to delete this before merging! Otherwise it will run on every PR (unless we want that as a test, but then you should change the comment. 😉).